sklearn/plot/_confusion_matrix.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_confusion_matrix(values, classes, normalize=False,
```

Comment:
I feel like it would be easier to just provide ``y_true`` and ``y_pred`` and make ``classes`` optional, so people don't have to import two functions?